roles/manage-ec2-instances/tasks/provision.yml

```diff
@@ -652,3 +693,8 @@
     src: instances.txt.j2
     dest: "{{ playbook_dir }}/workshops/{{ name_prefix }}/{{ user_prefix }}{{ item }}-instances.txt"
   with_sequence: count="{{ user_count }}"
+
+# - name: Configure external tower if needed
```

Comment:
I assume this is for a new feature down the road?

Reply:
yes; i wanted it to build out the stuff in my tower instance for some workshop management...